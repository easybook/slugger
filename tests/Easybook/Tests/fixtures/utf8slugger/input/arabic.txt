؆؇؈؉؊؋،؍؎؏
ءآأؤإئابةتثجحخد
ذرزسشصضطظعغػؼؽؾؿ
فقكلمنهوىي
٠١٢٣٤٥٦٧٨٩٪٫٬٭ٮٯ
ٰٱٲٳٴٵٶٷٸٹٺٻټٽپٿ
ڀځڂڃڄڅچڇڈډڊڋڌڍڎڏ
ڐڑڒړڔڕږڗژڙښڛڜڝڞڟ
ڠڡڢڣڤڥڦڧڨکڪګڬڭڮگ
ڰڱڲڳڴڵڶڷڸڹںڻڼڽھڿ
ۀہۂۃۄۅۆۇۈۉۊۋیۍێۏ
ۑےۓ۔ەۖ
ۮۯ
۰۱۲۳۴۵۶۷۸۹ۺۻۼ۽۾ۿ
ݐݑݒݓݔݕݖݗݘݙݚݛݜݝݞݟ
ݠݡݢݣݤݥݦݧݨݩݪݫݬݭݮݯ
ݰݱݲݳݴݵݶݷݸݹݺݻݼݽݾݿ