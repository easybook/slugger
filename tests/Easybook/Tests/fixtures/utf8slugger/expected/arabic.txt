
aawayabttthjhkhd
dhrzsshsdtzgh
fqklmnhwyy
0123456789
awyp
ch
zhs
vkngg

vy


0123456789



